image/image.go

```diff
@@ -219,6 +219,26 @@ func (c *Client) PutImage(ctx context.Context, userAuthToken, serviceAuthToken,
 	return
 }
 
+// PublishImage triggers an image publishing
+func (c *Client) PublishImage(ctx context.Context, userAuthToken, serviceAuthToken, collectionID, imageID string) (err error) {
+
+	uri := fmt.Sprintf("%s/images/%s/publish", c.url, imageID)
+
+	clientlog.Do(ctx, "updating instance import_tasks", service, uri)
+
+	resp, err := c.doPutWithAuthHeaders(ctx, userAuthToken, serviceAuthToken, collectionID, uri, []byte{})
```

Comment:
Should this be updated to use post now?

Reply:
thanks! I forgot to change it this morning (after changing image API).